docs/resources/widgets/pages/how-tos.md

```diff
@@ -0,0 +1,481 @@
+---
+title: How-to
+sidebar_position: 2
+toc_max_heading_level: 5
+---
+
+#
+
+FlutterFlow allows you to easily create new pages using the Page Selector
+button, which is available in the Widget Tree menu (Tab+2). This will help you
+quickly start and add new pages to your app.
+
+![page-selector.png](imgs%2Fpage-selector.png)
+
+## Create an empty page
+
+To create an empty FlutterFlow page from scratch, follow the steps:
+
+<iframe src="https://demo.arcade.software/rtOPRXVUByUAXxUynd8T?embed&show_copy_link=true" title="app.flutterflow.io/authentication" frameborder="0" loading="lazy" webkitallowfullscreen mozallowfullscreen allowfullscreen allow="clipboard-write" width="100%" height="600"></iframe>
+
+## Create a page from a popular template
+
+FlutterFlow simplifies the process of page creation by offering a variety of
+popular template use cases. These templates provide a basic structure for your
+pages, which you can quickly customize with your own styling, widgets, and text.
+
+To utilize a template from FlutterFlow, follow these steps:
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
```

Comment:
There is some weird spacing after the arcades on this page. Seems like we need some padding or something